pkg/dcgmexporter/parser.go

```diff
@@ -52,15 +52,15 @@ func ExtractCounters(c *Config) ([]Counter, []Counter, error) {
 			logrus.Fatal(err)
 		}
 	} else {
-		err = fmt.Errorf("no configmap data specified")
+		logrus.Infof("No configmap data specified")
 	}
 
-	if err != nil {
-		logrus.Infof("%v, falling back to metric file %s", err, c.CollectorsFile)
+	if err != nil || c.ConfigMapData == undefinedConfigMapData {
+		logrus.Infof("Falling back to metric file '%s'", c.CollectorsFile)
```

Comment:
When reviewing @glowkey and @nvvfedorov please pay special attention to this change.

Reply:
We need to add a unit test for this part of the code.

Reply:
The existing test case `TestExtractCounters` already covers this part of the code where it reads from the `prefix-` file created by the test case and K8s config map is not supplies. 

Unless you have a specific test case in mind.